### ISSUE TITLE
Screen Reader Doesn't Read Learning Objectives  

### ISSUE THREAD

**Issue by klbarnes20** [datacarpentry/R-ecology-lesson#612]
Hi everyone, 

I am going through this lesson to learn about using R and I stumbled on a potential issue. I often use a screen reader when reading a lot online. I discovered that for this curriculum, my screen reader doesn't recognize the Learning Objectives in the gray box at the top of the lessons. I am using this chrome extension: https://chrome.google.com/webstore/detail/read-aloud-a-text-to-spee/hdhinadidafjejdhmfkjgnolgimiaplp?hl=en 

I'm not sure if this problem is unique to my extension or a broader problem, but I thought I would bring it up as a potential accessibility issue. 

Thanks!

**Comment by ErinBecker** [comment]
Thank you for this issue @klbarnes20. @zkamvar and @fmichonneau are working on a new template prototype and including an accessibility review in the process. I'm pinging them here so they take this issue into consideration during the design phase.

**Comment by fmichonneau** [comment]
@klbarnes20 that's interesting. It looks like this extension recognizes the learning objective in some episodes but not others. It'd be interesting to investigate a little further but I don't see anything specific in the source that would explain the difference.

**Comment by Teebusch** [comment]
Using the extension, it seems that the objectives are only read for "Before we Start" and "Manipulating Data". The learning objects themselves are indeed always formatted alike (as far as I can tell). To me it seems like the issue may be with different levels of headings being used throughout the document, but I can't pinpoint it perfectly.
